gallery/src/components/NxSearchDropdown/NxSearchDropdownPage.tsx

```diff
@@ -0,0 +1,228 @@
+    nxSearchDropdownErrorExampleCode = require('./NxSearchDropdownErrorExample?raw'),
+    nxSearchDropdownLongErrorExampleCode = require('./NxSearchDropdownLongErrorExample?raw'),
+    nxSearchDropdownDisabledExampleCode = require('./NxSearchDropdownDisabledExample?raw');
+
+const NxSearchDropdownPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        A search text input that shows matching results in a dropdown for the user to select. Most commonly used
+        when the search requires a backend query.
+      </NxP>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>Props</NxH3>
+        </NxTile.SubsectionHeader>
+        <NxTable>
+          <NxTable.Head>
+            <NxTable.Row>
+              <NxTable.Cell>Prop</NxTable.Cell>
+              <NxTable.Cell>Type</NxTable.Cell>
+              <NxTable.Cell>Required</NxTable.Cell>
+              <NxTable.Cell>Default</NxTable.Cell>
+              <NxTable.Cell>Description</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Head>
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>searchText</NxCode></NxTable.Cell>
+              <NxTable.Cell>string</NxTable.Cell>
+              <NxTable.Cell>true</NxTable.Cell>
+              <NxTable.Cell></NxTable.Cell>
+              <NxTable.Cell>The current value of the search box</NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>onSearchTextChange</NxCode></NxTable.Cell>
+              <NxTable.Cell>Function (string =&gt; void)</NxTable.Cell>
+              <NxTable.Cell>true</NxTable.Cell>
+              <NxTable.Cell></NxTable.Cell>
+              <NxTable.Cell>
+                Callback that fires when the user changes the search text. The handler passed in for this prop should
+                update the <NxCode>searchText</NxCode> prop. The handler receives the new search text value as its
+                argument.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>onSearch</NxCode></NxTable.Cell>
+              <NxTable.Cell>Function (string =&gt; void)</NxTable.Cell>
+              <NxTable.Cell>true</NxTable.Cell>
+              <NxTable.Cell></NxTable.Cell>
+              <NxTable.Cell>
+                Callback that fires when a new search query should be performed. The <em>trimmed</em> value of the
+                filter text is passed as an argument. This callback is executed whenever the user changes
+                the trimmed value of the filter input, whenever the error alert's Retry button is clicked, and whenever
+                the component gains focus while in an error state (which is intended to automatically trigger a retry
+                attempt. The handler passed in for this prop should, in addition to performing the search, update
```

Comment:
Missing closing bracket.
```suggestion
                filter text is passed as an argument. This callback is executed whenever the user changes
                the trimmed value of the filter input, whenever the error alert's Retry button is clicked, and whenever
                the component gains focus while in an error state (which is intended to automatically trigger a retry
                attempt). The handler passed in for this prop should, in addition to performing the search, update
```